packages/lib/raqb/resolveQueryValue.ts

```diff
@@ -0,0 +1,252 @@
+import logger from "@calcom/lib/logger";
+import type { dynamicFieldValueOperands, dynamicFieldValueOperandsResponse } from "@calcom/lib/raqb/types";
+import type { AttributesQueryValue } from "@calcom/lib/raqb/types";
+import type { Attribute } from "@calcom/lib/service/attribute/server/getAttributes";
+
+import { caseInsensitive } from "./utils";
+
+// Type for JSON values that can be in the query
+type JsonValue = string | number | boolean | null | JsonValue[] | { [key: string]: JsonValue };
+
+const moduleLogger = logger.getSubLogger({ prefix: ["raqb/resolveQueryValue"] });
+
+/**
+ * Replace attribute option Ids with the attribute option label(compatible to be matched with form field value)
+ */
+const replaceAttributeOptionIdsWithOptionLabel = ({
+  queryValue,
```

Comment:
This fn came from raqbUtils with the difference that it now accepts _queryValue_ instead of _queryValueString_. It  builds _queryValueString_ itself